README.md

```diff
@@ -51,6 +52,7 @@ Increase `ulimit` to avoid such errors. For example,
 * [JBeret-dev Mailing List](https://lists.jboss.org/mailman/listinfo/jberet-dev)
 * [WildFly Forum](https://groups.google.com/g/wildfly)
 * [JBeret Forum](https://developer.jboss.org/en/jberet/)
+* [Jakarta Batch Expert Group Discussion](https://java.net/projects/jbatch/lists/public/archive)
```

Comment:
I think we should refer users to the latest location: https://github.com/eclipse-ee4j/batch-api/issues

Reply:
Changes done.